### PR TITLE
Updated the references to the digest library

### PR DESCRIPTION
**Pull request by MihaiBojin**
## Description

Please include a summary of the fix/feature/change, including any relevant motivation and context.

Link to any related issue(s): 

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Required Checklist:

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run `make fmt` and formatted my code

### CODE REVIEW

File: mongodbatlas/doc.go [hunk 1]
```diff
@@ -36,30 +36,30 @@ can be used as a starting point.
...
-creating a new client, pass an http.Client that can handle authentication for
-you. The easiest and recommended way to do this is using the https://github.com/Sectorbob/mlab-ns2/gae/ns/digest
-library, but you can always use any other library that provides an http.Client.
-If you have a private and public API token pair, you can
-use it with the digest library using:
+creating a new client, pass an http.Client that can handle Digest Access authentication for
+you. The easiest way to do this is using the [digest](https://github.com/mongodb-forks/digest)
+library, but you can always use any other library that provides an `http.Client`.
+If you have a private and public API token pair, you can use it with the digest library using:
+```go
```
Comment (gssbzn) [inline mongodbatlas/doc.go:43]: go doc does not support markdown, you need to keep this indented so it's rendered as code

File: mongodbatlas/doc.go [hunk 2]
```diff
@@ -36,30 +36,30 @@ can be used as a starting point.
 Authentication
 
 The mongodbatlas library does not directly handle authentication. Instead, when
-creating a new client, pass an http.Client that can handle authentication for
-you. The easiest and recommended way to do this is using the https://github.com/Sectorbob/mlab-ns2/gae/ns/digest
-library, but you can always use any other library that provides an http.Client.
-If you have a private and public API token pair, you can
-use it with the digest library using:
+creating a new client, pass an http.Client that can handle Digest Access authentication for
+you. The easiest way to do this is using the [digest](https://github.com/mongodb-forks/digest)
```
Comment (gssbzn) [inline mongodbatlas/doc.go:40]: same, you can't use markdown here

Comment (gssbzn) [reply mongodbatlas/doc.go:40]: this one still stands, `[text](url)` will render verbatim (WYSIWYG)

File: README.md [hunk 3]
```diff
@@ -31,17 +31,16 @@ can be used as a starting point.
...
 
 The mongodbatlas library does not directly handle authentication. Instead, when
-creating a new client, pass an http.Client that can handle authentication for
-you. The easiest and recommended way to do this is using the [digest](https://github.com/Sectorbob/mlab-ns2/gae/ns/digest)
+creating a new client, pass an http.Client that can handle Digest Access authentication for
+you. The easiest way to do this is using the [digest](https://github.com/mongodb-forks/digest)
 library, but you can always use any other library that provides an `http.Client`.
-If you have a private and public API token pair, you can
-use it with the digest library using:
+If you have a private and public API token pair, you can use it with the digest library using:
```
Comment (themantissa) [inline README.md:37]: Suggest adding a link to how to create the pair - we get a lot of user comments about how it's hard to figure this out.  https://docs.atlas.mongodb.com/configure-api-access/ is a pretty solid page as it has the UI instructions and links to the API as well.

**Pull request merged**